tests/base/symmetric_tensor_39.cc

```diff
@@ -0,0 +1,63 @@
+#include <iostream>
+#include <fstream>
+
+#include <deal.II/base/symmetric_tensor.h>
+#include <deal.II/base/vectorization.h>
+
+using namespace dealii;
+
+int main()
+{
+  std::string logname = "output";
+  std::ofstream logfile(logname);
+
+#if defined DEAL_II_COMPILER_VECTORIZATION_LEVEL
+  std::cout << "Vectorization preprocessor variable found!" << std::endl;
+#endif
+
+#if DEAL_II_COMPILER_VECTORIZATION_LEVEL == 0
+  std::cout << "Value of DEAL_II_COMPILER_VECTORIZATION_LEVEL represents no vectorization" << std::endl;
+#elif DEAL_II_COMPILER_VECTORIZATION_LEVEL == 1
+  std::cout << "Value of DEAL_II_COMPILER_VECTORIZATION_LEVEL represents SSE" << std::endl;
+#elif DEAL_II_COMPILER_VECTORIZATION_LEVEL == 2
+  std::cout << "Value of DEAL_II_COMPILER_VECTORIZATION_LEVEL represents AVX" << std::endl;
+#else
+  std::cout << "Value of DEAL_II_COMPILER_VECTORIZATION_LEVEL not in {0,1,2}" << std::endl;
+#endif
```

Comment:
Could you please remove these lines? I do not see a big value in them. Moreover, you are missing `DEAL_II_VECTORIZATION_LEVEL==3` (AVX-512) here (for the moment only Intel Knights Landing Xeon Phi, but soon more ubiquitous among server processors).